site-content/source/modules/ROOT/pages/development/documentation.adoc

```diff
@@ -19,7 +19,7 @@ instead of visual editors (which some people would call good news).
 Antora processes http://www.asciidoc.org[Ascidoc], the markup language used to generate our documentation.
 Markup languages allow you to format text using certain syntax elements.
 Your document structure will also have to follow specific conventions.
-Feel free to take a look at http://cassandra.apache.org/docs[existing documents] to get a better idea how we structure our documents.
+Feel free to take a look at link:/doc/latest/[existing documents] to get a better idea how we structure our documents.
```

Comment:
i don't think this is intended to go to /doc/latest/ but /doc/ but it's supposed to show you the different versions we have.

e.g. see https://cassandra.apache.org/doc/

Reply:
So drop /latest/?

Reply:
willdo

Reply:
done